cli/monitor/monitor.go

```diff
@@ -87,10 +97,63 @@ func runMonitorCmd(cmd *cobra.Command, args []string) {
 	}
 	defer tty.Close()
 
+	configuration := &rpc.MonitorPortConfiguration{}
+	if len(configs) > 0 {
+		resp, err := monitor.EnumerateMonitorPortSettings(context.Background(), &rpc.EnumerateMonitorPortSettingsRequest{
+			Instance: instance,
+			Port:     port.ToRPC(),
+			Fqbn:     "",
+		})
+		if err != nil {
+			feedback.Error(err)
+			os.Exit(errorcodes.ErrGeneric)
+		}
+		settings := resp.GetSettings()
+		for _, config := range configs {
+			split := strings.SplitN(config, "=", 2)
+			k := ""
+			v := config
+			if len(split) == 2 {
+				k = split[0]
+				v = split[1]
+			}
+
+			var setting *rpc.MonitorPortSettingDescriptor
+			for _, s := range settings {
+				if k == "" {
+					if contains(s.EnumValues, v) {
+						setting = s
+						break
+					}
+				} else {
+					if strings.EqualFold(s.SettingId, k) {
+						if !contains(s.EnumValues, v) {
+							feedback.Error(tr("invalid port configuration value for %s: %s", k, v))
+							os.Exit(errorcodes.ErrBadArgument)
+						}
+						setting = s
+						break
+					}
+				}
+			}
+			if setting == nil {
+				feedback.Error(tr("invalid port configuration: %s", config))
+				os.Exit(errorcodes.ErrBadArgument)
+			}
+			configuration.Settings = append(configuration.Settings, &rpc.MonitorPortSetting{
+				SettingId: setting.SettingId,
+				Value:     v,
+			})
+			if !silent {
+				feedback.Print(fmt.Sprintf("Set %s to %s", setting.SettingId, v))
```

Comment:
```suggestion
				feedback.Print(tr("Set %s to %s", setting.SettingId, v))
```

Reply:
Since the original `Set %s to %s` doesn't give enough context for translation I changed to `Monitor port settings:`